frontend/public/components/RBAC/bindings.jsx

```diff
@@ -107,13 +108,43 @@ const menuActions = ({subjectIndex, subjects}, startImpersonate) => {
   return actions;
 };
 
-const Header = props => <ListHeader>
-  <ColHead {...props} className="col-md-3 col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-md-3 col-sm-4 hidden-xs" sortField="roleRef.name">Role Ref</ColHead>
-  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="subject.kind">Subject Kind</ColHead>
-  <ColHead {...props} className="col-md-2 hidden-sm hidden-xs" sortField="subject.name">Subject Name</ColHead>
-  <ColHead {...props} className="col-md-2 col-sm-4 col-xs-6" sortField="metadata.namespace">Namespace</ColHead>
-</ListHeader>;
+const tableColumnClasses = [
+  classNames('pf-m-3-col-on-lg', 'pf-m-4-col-on-md', 'pf-m-6-col-on-sm'),
```

Comment:
It looks like these classes don't match what was there before. Was that intentional?

Reply:
yes... i've noted all breakpoint modifications in the checklist above.

> fix slight overflow on md breakpoint due to sort icon (Alert Managers, Service Accounts, Role Bindings, Config Maps, Secrets), xl breakpoint (Horizontal Pod AutoScalers) (see priley86@0d106ea)

let me know if you think any of these breakpoint modifications are undesired. These were just a few exceptional cases though where things did not translate 1:1 due to slight spacing overflows.

Reply:
> yes... i've noted all breakpoint modifications in the checklist above.

Thanks, no concerns. Just making sure it was intended. Missed you comment above.